packet/client.go

```diff
@@ -48,7 +48,7 @@ func NewClient(consumerToken, authToken string, baseURL *url.URL) (*Client, erro
 	dataModelVersion := os.Getenv("DATA_MODEL_VERSION")
 	switch dataModelVersion {
 	case "1":
-		hg, err = tinkClient.NewTinkerbellClient()
+		hg, err = tinkClient.TinkerbellHWClient()
```

Comment:
I think it's better to write out the word `Hardware`. It's easier to read and more consistent with `TinkerbellWorkflowClient` in my opinion.

Reply:
Agreed, good point Kelly!

Reply:
done